docs/using-kontena/loadbalancer.md

```diff
@@ -74,8 +76,8 @@ These environment variables configure the load balancer itself.
 
 * `KONTENA_LB_HEALTH_URI` - URI at which to enable Kontena Load Balancer level health check endpoint. Returns `200 OK` when Kontena Load Balancer is functional.
 * `STATS_PASSWORD` - The password for accessing Kontena Load Balancer statistics.
-* `SSL_CERTS` - SSL certificates to be used. See more at [SSL Termination](loadbalancer#ssl-termination).
-* `KONTENA_LB_SSL_CIPHERS` - SSL Cipher suite used by the loadbalancer when operating in SSL mode. See more at [SSL Ciphers](loadbalancer#configuringcustomsslciphers)
+* `SSL_CERTS` - SSL certificates to be used. See more at [SSL Termination](loadbalancer.md#ssl-termination).
+* `KONTENA_LB_SSL_CIPHERS` - SSL Cipher suite used by the loadbalancer when operating in SSL mode. See more at [SSL Ciphers](.md#configuringcustomsslciphers)
```

Comment:
`[SSL Ciphers](.md#configuringcustomsslciphers)` looks like odd to me. I think the link be `loadbalancer.md#configuringcustomsslciphers`?